### PR TITLE
#155820645 Create layout file for detail activity

### PR DESCRIPTION
**Pull request by wycliffkas**
#### What does this PR do?
Creates a layout file for a detailed activity
#### Description of Task to be completed?
Create a layout for displaying details of a single developer
#### What are the relevant pivotal tracker stories?
[ #155820645](https://www.pivotaltracker.com/story/show/155820645)

### CODE REVIEW

File: app/src/main/res/layout/developer_details.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                <TextView
+                    android:id="@+id/textView3"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="72dp"
+                    android:layout_marginLeft="72dp"
+                    android:layout_marginTop="13dp"
+                    android:layout_marginEnd="72dp"
+                    android:layout_marginRight="72dp"
+                    android:text="https://github.com/wycliffkas"
```
Comment (huxaiphaer) [inline app/src/main/res/layout/developer_details.xml]: This should be declared in the `strings.xml`.

File: app/src/main/res/layout/developer_details.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <TextView
+                    android:id="@+id/textView2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="117dp"
+                    android:layout_marginLeft="117dp"
+                    android:layout_marginTop="52dp"
+                    android:layout_marginEnd="117dp"
+                    android:layout_marginRight="117dp"
+                    android:text="Wycliff Kasirye"
```
Comment (huxaiphaer) [inline app/src/main/res/layout/developer_details.xml]: This should be declared in the strings.xml.

File: app/src/main/res/layout/developer_details.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    app:layout_constraintHorizontal_bias="0.0"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:srcCompat="@drawable/user" />
+
+                <TextView
+                    android:id="@+id/textView2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="117dp"
```
Comment (huxaiphaer) [inline app/src/main/res/layout/developer_details.xml]: Declare these sizes in the `dimens.xml`

**Review by huxaiphaer — COMMENTED** [review]
Work on the feedback I have left.

**Comment by wycliffkas** [comment]
> Work on the feedback I have left.

Noted. Thank you,  am going to work on the feedback

**Review by huxaiphaer — APPROVED** [review]
LGTM

**Pull request merged**